prov/rxm/src/rxm_ep.c

```diff
@@ -1499,9 +1499,7 @@ static int rxm_ep_msg_cq_open(struct rxm_ep *rxm_ep, enum fi_wait_obj wait_obj)
 
 	return 0;
 err:
-	ret = fi_close(&rxm_ep->msg_cq->fid);
-	if (ret)
-		FI_WARN(&rxm_prov, FI_LOG_EP_CTRL, "Unable to close msg CQ\n");
+	fi_close(&rxm_ep->msg_cq->fid);
```

Comment:
Shouldn't we keep the warning?

Reply:
honestly i don't think it's worth it for this error path, but I can add it back if you want

Reply:
Okay. Ideally `fi_close()` shouldn't fail here so I think we should be good.